### PR TITLE
32 Spawn card

### LINKED ISSUE #32: Card spawning

### PR DESCRIPTION
**Pull request by njramage**
Add missing things in game scene
Stop card from falling if dragged
Make cards fall

Tasks:
- Need to chnage the colour of the indicators
- Need player feedback on correct matches
- Some magic in spawn limits

Note:
- When using a point always need to convert to world space

### CODE REVIEW

**Comment by njramage** [comment]
Worth play testing

File: Assets/Scripts/GameManager.cs
Comment (TyeBryant) [inline Assets/Scripts/GameManager.cs:1]: Seems odd to have just added usings to a file

File: Assets/Scripts/UIManager.cs
Comment (TyeBryant) [inline Assets/Scripts/UIManager.cs:1]: Same again re-usings

Comment (njramage) [reply Assets/Scripts/UIManager.cs:1]: I'll remove this. I was playing with this class the using would have been added during that by the editor and I didn't check it when making the commit

**Review by HarrisonShort — APPROVED** [review]

**Pull request merged**